src/AspNet.Security.OAuth.Validation/Events/AuthenticationFailedContext.cs

```diff
@@ -2,7 +2,7 @@
 using Microsoft.AspNetCore.Http;
 using System;
 
-namespace AspNet.Security.OAuth.Validation.Events
```

Comment:
Hmm... I would prefer directory structure to correspond to namespaces.
I cloned your project and `Events` folder is still there, which is why I'm for having the `.Event` namespace.